jkanban.js

```diff
@@ -508,6 +508,9 @@ var dragula = require('dragula');
     }
 
     function __onclickHandler (nodeItem, clickfn) {
+      if (typeof self.options.click !== 'function') {
```

Comment:
See that browser event is cancelled by **preventDefault** method. So if we have a property to control if jKanban has to propagate click or context this if will be simpler like ´if (!self.ownedHandlers.includes('click')) { return }`